index.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```

Comment:
@Akshay8122 I see merge conflict marker here. Resolve this.

Reply:
i have fixed it